src/tools/pub/cmd/pub-add.md

```diff
@@ -37,12 +38,32 @@ $ dart pub add foo:'>=2.0.0 <3.0.1'
 ```
 
 If `dart pub add <package>:<constraint>` is an existing dependency,
-it will update the constraint.
+it will instead update the existing constraint.
```

Comment:
This is a huge nitpick/overthink,, so you don't have to take it, but I thought this wording could be a little more explicit (the original too, which I wrote, so not your fault). 

I think your edit adding "existing" moves in the right direction, but I'm still concerned saying it "updates the constraint" could be mistakenly conflated with "progresses to the next latest version", when really it's updating the pubspec entry, and the "new" constraint could be anything, even an older version. Maybe for the sake of overly-clarifying we could say:

> If `<package>` (or `<package>:<constraint>`) is an existing dependency in `pubspec.yaml`,
> `dart pub add <package>:<constraint>` updates the dependency's
> constraint to the one specified in the command.

Or maybe it'd be more useful to make this more action-driven, like:
> You can change an existing dependency's constraint in your `pubspec.yaml`
> by running `dart pub add <package>:<constraint>` again. 

And if you decide to leave it as your first edit, I'd remove "instead": "...it will update the existing constraint."

Reply:
What you suggested makes a lot more sense, thanks for the thoughtful suggestion. I went with a derivative of your first suggestion, let me know how that sounds :)

Reply:
Everything looks good! Double approved :)